pdata/ptrace/generated_status.go

```diff
@@ -72,6 +61,17 @@ func (ms Status) SetMessage(v string) {
 	ms.orig.Message = v
 }
 
+// Code returns the code associated with this Status.
+func (ms Status) Code() StatusCode {
```

Comment:
same here - looks like the code is just being regenerated out of order, can this be addressed to reduce the PR's impact?

Reply:
The problem is that we want the fields in the proto defined order to ensure it is easier to follow the IDs :(